neon_core/configuration/mark_2/neon.yaml

```diff
@@ -42,14 +42,19 @@ hotwords:
     active: false
   hey_neon:
     model_folder: /home/neon/.local/share/neon/vosk-model-small-en-us-0.15
+    active: false
```

Comment:
This should default to `active: true`; currently that's not set, but the default `configuration/neon.yaml` specifies this as the default wake word: https://github.com/NeonGeckoCom/NeonCore/blob/99288a32118175da12ce779da48a145cad06bbb6/neon_core/configuration/neon.yaml#L80